test_unstructured_ingest/expected-structured-output/azure/Core-Skills-for-Biomedical-Data-Scientists-2-pages.pdf.json

```diff
@@ -17,7 +17,8 @@
           "remote_file_path": "abfs://container1/"
         },
         "date_created": "1678440764.0",
-        "date_modified": "1678440764.0"
+        "date_modified": "1678440764.0",
+        "filesize_bytes": 41164
```

Comment:
i think i'm neutral either way, but maybe worth adding `filesize_bytes` as `--metadata_exclude` args to avoid the fixture churn.